app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala

```diff
@@ -73,7 +73,8 @@ object Section {
     cyaPage: Option[CheckYourAnswersPage] = None,
     fields: Option[NonEmptyList[FormComponent]] = None,
     pageIdToDisplayAfterRemove: Option[PageId] = None,
-    notRequiredIf: Option[IncludeIf]
+    notRequiredIf: Option[IncludeIf],
+    declarationSection: Option[DeclarationSection] = None
```

Comment:
Shouldn't this live inside of `CheckYourAnswersPage`?

According to ticket it cannot exists by itself without cya page.